client/src/App.js

```diff
@@ -18,6 +20,13 @@ function App() {
         <Route path = '/user/id/events' element = {<UserEvents/>}/>
```

Comment:
same as the other comment (id -> :id) to be able to dynamically select the user. The current implementation would match the route `/user/id/events` but not `/user/2/events`